modules/SDL.lua

```diff
@@ -178,11 +216,14 @@ end
 --- Update SDL log4cxx.properties in order SDL will be able to write logs through Telnet
 local function updateSDLLogProperties()
   if config.storeFullSDLLogs then
-    local pathToFile = config.pathToSDL .. "/log4cxx.properties"
-    local f = io.open(pathToFile, "r")
-    local content = f:read("*all")
-    f:close()
-
+    if config.remoteConnection.enabled then
+      local content = ATF.remoteUtils.file:GetFileContent(config.pathToSDL .. "/","log4cxx.properties")
```

Comment:
@AStarodub Please take a look on return values of `ATF.remoteUtils.file:GetFileContent` function
```
-- @treturn boolean Return true in case of success
-- @treturn string Content of file
function RemoteFileUtils.mt.__index:GetFileContent(remotePathToFile, fileName)
```

Reply:
done:changed according to remote utils specification